_includes/footer.html

```diff
@@ -12,7 +12,7 @@
           <a class="sitemap--list__link sitemap--list__link--parent" href="https://twitter.com/kleesymex">Twitter</a>
         </li>
         <li class="sitemap__item sitemap__item--doxygen">
-          <a class="sitemap--list__link sitemap--list__link--parent" href="">Doxygen</a>
+          <a class="sitemap--list__link sitemap--list__link--parent" href="https://klee.github.io/releases/docs/v2.3/doxygen/html">Doxygen</a>
```

Comment:
Shouldn't it be: `href="{{ site.baseurl }}/releases/docs/v2.3/doxygen/html"`?

Reply:
Good catch, but no.  This is what I had initially, but `site.baseurl` is set to `https://klee.github.io/releases/docs/v2.3/` for the 2.3 release so the link would be incorrect. 
It's obviously not a very elegant solution, but I think the right one given the special handling of Doxygen now.

Reply:
So: `href="{{ site.baseurl }/doxygen/html"`?

Reply:
But this won't work for master then.